sdk/maps/azure-maps-geolocation/README.md

```diff
@@ -52,7 +52,26 @@ geolocation_client = MapsGeolocationClient(
 )
 ```
 
-#### 2. Authenticate with an Azure Active Directory credential
+#### 2. Authenticate with a SAS Credential
+
+You can authenticate with your generated SAS token.
+Once the SAS token is created, set the value of the token as environment variable: `AZURE_SAS_TOKEN`.
+Then pass an `AZURE_SAS_TOKEN` as the `credential` parameter into an instance of AzureSasCredential.
```

Comment:
The description in this paragraph is incomplete. We need to tell user to install `azure-mgmt-maps` and `azure-identity`:
```bash
pip install azure-mgmt-maps
pip install azure-identity
```

And tell them how to get the correct `AZURE_SAS_TOKEN` value using `list_sas` method from `azure-mgmt-maps`.

Reply:
Same for JAVA SDK.